src/simtools/simulator.py

```diff
@@ -460,62 +461,53 @@ def _fill_results(self, file, run_number):
             run number
 
         """
-        keys = ["output", "sub_out", "log", "input", "hist", "corsika_log"]
+        keys = ["output", "sub_out", "log", "input", "hist", "corsika_log", "event_data"]
         results = {key: [] for key in keys}
 
+        def get(name, **kwargs):
```

Comment:
`get` is too general a name for this function I think. Maybe `get_file_name_str`?